.github/workflows/automerge.yml

```diff
@@ -0,0 +1,13 @@
+name: Auto Merge Dependency Updates
+
+on:
+  - pull_request
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: tjenkinson/gh-action-auto-merge-dependency-updates@1ff3f19
```

Comment:
https://github.com/dyue/hls.js/runs/4525499357?check_suite_focus=true
Looks like the short form SHA has been deprecated. We will need to provide the long form:
```@ 1ff3f19509b1965cf1753c294c55e8cdb0528178```

https://github.blog/changelog/2021-01-21-github-actions-short-sha-deprecation/